modules/libcom/RTEMS/setBootConfigFromNVRAM.c

```diff
@@ -23,12 +21,45 @@
 #include <epicsStdio.h>
 #include <epicsString.h>
 #include <envDefs.h>
+#include <stdio.h>
 
 char *env_nfsServer;
 char *env_nfsPath;
 char *env_nfsMountPoint;
 
-extern char* rtems_bsdnet_bootp_cmdline;
+#ifndef RTEMS_LEGACY_STACK
```

Comment:
Create macros here and in a `#else` case to abstract the accesses to these members below, reducing the number of `#if` conditionals further down.